README.md

```diff
@@ -115,6 +116,81 @@ testing. To test:
 
 Testing is only supported on Linux with bash.
 
+## Post-Quantum Algorithms
+
+Starting with wolfSSL version 5.0.1 and nginx version 1.21.4, You can now enable the integration of liboqs in wolfSSL thus enabling post-quantum algorithms for your HTTPS connections over TLS 1.3. In this case, the web server will be nginx with wolfSSL and the web client will be curl with wolfSSL.
+
+First, you will need to build the OpenQuantumSafe group's liboqs and their fork of OpenSSL to generate the certificate chain that uses the post-quantum FALCON signature scheme. Instructions for that are in wolfSSL git repoitory's INSTALL file. Note that when you generate your certificates, you will need to add your IP address as a subject alternative name. See here for more details: https://www.openssl.org/docs/manmaster/man5/x509v3_config.html
+
+When building wolfSSL, you will need to add a couple extra flags:
+
+```
+./configure --enable-nginx --with-liboqs --enable-all
+make all
+make check
+sudo make install
+```
+
+NOTE: `--enable-all` is for curl.
+
+Now, you can continue on with the instructions for building nginx above. Once that is done, you'll need to build curl. You will need curl 7.80.0 or later. After unpacking curl, do the following:
+
+```
+./configure --with-wolfssl=/usr/local
+make all
+sudo make install
+```
+
+This will install the curl executable in the default location: `/usr/local/bin/curl`.
+
+Now that all the software is built and installed, you will need to add a section in the nginx.conf file to enable TLS 1.3 and use the correct certificates. Edit `/usr/local/nginx/conf/nginx.conf`. Nginx's install process should have put a default version there. Search for the section with the title `HTTPS server` and replace that section with the following:
+
+```
+    server {
+        listen                    443 ssl;
+        server_name               localhost;
+
+        ssl_certificate           /path/to/falcon_level5_entity_cert.pem;
```

Comment:
Do you need to link to the instructions for setting up `falcon_level5_entity_cert.pem` and `falcon_level5_entity_key.pem`?

Reply:
I mention the INSTALL file in wolfssl earlier which has those instructions.